site/_docs/howto.md

```diff
@@ -99,6 +99,10 @@ $ ./gradlew spotlessCheck checkstyleAll # report code style violations
 
 You can use `./gradlew assemble` to build the artifacts and skip all tests and verifications.
 
+You can use `./gradlew :{module}:{command}` pattern to run a command in a specific
```

Comment:
Does it really help?

Isn't it better to put a link to the documentation?
https://docs.gradle.org/current/userguide/what_is_gradle.html#five_things + 
https://docs.gradle.org/current/userguide/intro_multi_project_builds.html

I think it covers that pretty well.

Reply:
You are right. The links are much better.

Reply:
@vlsi updated PR to add two links only.